hyperspy/gui/egerton_quantification.py

```diff
@@ -145,7 +145,8 @@ def bg_to_plot(self, axes_manager=None, fill_with=np.nan):
         return to_return
 
     def span_selector_changed(self):
-        if (self.ss_left_value is np.nan) or (self.ss_right_value is np.nan):
+        if self.ss_left_value is np.nan or self.ss_right_value is np.nan or\
+                self.ss_right_value <= self.ss_left_value:
```

Comment:
Won't hurt, but if this was actually an issue, this is the wrong place to fix it :)